cluster-provision/images/vm-image-builder/alpine-cloud-init/create-image.sh

```diff
@@ -15,8 +15,8 @@ fi
 docker run --rm --platform=$PLATFORM -v /lib/modules:/lib/modules -v /dev:/dev --privileged -v $(pwd):$(pwd):z alpine ash -c "cd $(pwd) &&
 ./alpine-make-vm-image \
 	--image-format qcow2 \
-	--image-size 144M \
-    --branch v3.15 \
+	--image-size 200M \
```

Comment:
Can the commit msg be expanded to explain why we are increasing the size of the image? is it due to installing openssh? or due to the bump?

Reply:
I saw some "no space left" issues starting it up at kubevirt/kubevirt. I will update PR message